### ISSUE TITLE
Gif support

### ISSUE THREAD

**Issue by FoxPride** [sskodje/ScreenRecorderLib#129]
Greetings, @sskodje! 

Is it possible to add gif container support so we can take screenshots directly into gif?

I found this [library](https://github.com/charlietangora/gif-h) but got stuck with converting directx image.

**Comment by sskodje** [comment]
I'll put it in the backlog as a feature request, but for now i have to feature freeze for next release, it was a lot more work than anticipated. Can i ask what your use case is? GIF does not work very well for high-resolution content, it can easily run in the hundreds of megabytes for a short clip.

**Comment by FoxPride** [comment]
Oh that's sad. Sometimes I don't need whole video so I decided to check if it's possible to create gifs and use it for short duration.

**Comment by FoxPride** [comment]
Maybe it's better to use apng or webp then?

**Comment by sskodje** [comment]
You can also record as slideshow and stich the .png or .jpg images into a GIF using GifBitmapEncoder or something like that, you don't need to fiddle with directx.

**Comment by FoxPride** [comment]
Yeah. It was my first guess and it works but I thought about better solution. I'll stick with slideshow option then, thanks a lot for great library!
